src/views/v2/tickets/item/quantity/add.php

```diff
@@ -14,8 +14,8 @@
  *
  * @version TBD
  *
- * @var $this Tribe__Tickets__Editor__Template
- * @var Tribe__Tickets__Ticket_Object $ticket The ticket object.
+ * @var	Tribe__Tickets__Editor__Template $this   Template object.
+ * @var	Tribe__Tickets__Ticket_Object    $ticket The ticket object.
```

Comment:
:no_entry_sign: Spaces must be used for mid-line alignment; tabs are not allowed

<sub>hash: 13c1694d16b2f2f2b550c06050b0bc42</sub>